howto/stream/exchange-oob-data.md

```diff
@@ -24,83 +24,111 @@ The following instructions guide you to exchange OOB data using a specific imple
+In the `service.py` file of the stream client sample project, replace `<session_id>` with the session ID retrieved above. Next, launch the web client by opening `https://<appliance_private_ip>:8080/<session_id>` in your browser. Once the web page is fully loaded, it establishes a WebRTC connection to the active session. After the WebRTC peer connection is successfully made, the `foo` data channel will be created on the Anbox runtime server side, in response to the client's request.
 
-At the same time, a number of Unix domain sockets are created under the `/run/user/1000/anbox/sockets` folder. They use the format `webrtc_data_<channel_name>` and represent the established communication bridge between a WebRTC client and the Anbox WebRTC platform. Those Unix domain sockets can be used by a service or daemon to:
+At the same time, a Unix domain socket is created under the `/run/user/1000/anbox/sockets` folder in the format of `webrtc_data_<channel_name>` (`webrtc_data_foo` in the example) and represent the established communication bridge between a WebRTC client and the Anbox runtime. This Unix domain socket can be used by a service or daemon to:
 
 - Receive data sent from a WebRTC client over the data channel and forward it to an Android application.
 - Receive data sent from an Android application and forward it to a WebRTC client over the data channel.
 
-A trivial example to simulate the data transmission between an instance and a WebRTC client is using the [`socat`](https://manpages.ubuntu.com/manpages/bionic/man1/socat.1.html) command to connect the Unix domain socket and perform bidirectional asynchronous data sending and receiving:
+To simulate data transmission between the Anbox runtime and the web client, you can use the [`socat`](https://manpages.ubuntu.com/manpages/bionic/man1/socat.1.html) command to connect the Unix domain socket and perform bidirectional asynchronous data sending and receiving:
 
 1. Connect the Unix domain socket:
 
    ```
    socat - UNIX-CONNECT:/run/user/1000/anbox/sockets/webrtc_data_foo
    ```
 
-1. After the Unix domain socket is connected, type a message and hit the `Enter` key:
+1. This command establishes a connection to the `webrtc_data_foo` Unix domain socket, allowing you to send and receive data directly through the established data channel between Anbox runtime and the web client. After the Unix domain socket is connected, type a message and hit the `Enter` key:
 
        hello world
 
-   The data is now sent from the Anbox WebRTC platform over the data channel to the WebRTC client.
-1. Observe that the message is displayed in the console of a web browser, responding to the message event:
+   The data is now sent from the Anbox runtime over the data channel to the web client.
+1. Observe that the message is displayed in the console of a web client, responding to the message event:
```

Comment:
What console do we mean here? If possible we should provide links to browser documentation.

Reply:
+1.